sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/jsonpatch/JsonPatchDocument.java

```diff
@@ -0,0 +1,210 @@
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Represents a JSON Patch document.
+ */
+public class JsonPatchDocument {
+    private static final ObjectMapper MAPPER = ((JacksonAdapter) JacksonAdapter.createDefaultSerializerAdapter())
+        .serializer();
+
+    private final ClientLogger logger = new ClientLogger(JsonPatchDocument.class);
+
+    private final List<JsonPatchOperation> operations;
+
+    /**
+     * Creates a new JSON Patch document.
+     */
+    public JsonPatchDocument() {
+        this.operations = new ArrayList<>();
+    }
+
+    /**
+     * Appends an "add" operation to this JSON Patch document.
+     * <p>
+     * If the {@code path} doesn't exist a new member is added to the object. If the {@code path} does exist the
+     * previous value is replaced. If the {@code path} specifies an array index the value is inserted at the specified.
+     * <p>
+     * See <a href="https://tools.ietf.org/html/rfc6902#section-4.1">JSON Patch Add</a> for more information.
+     *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * {@codesnippet com.azure.core.experimental.jsonpatch.JsonPatchDocument.appendAdd#String-String}
+     *
+     * @param path The path to apply the addition.
+     * @param rawJsonValue The raw JSON value to add to the path.
+     * @return The updated JsonPatchDocument object.
+     * @throws NullPointerException If {@code path} or {@code rawJsonValue} is null.
+     */
+    public JsonPatchDocument appendAdd(String path, String rawJsonValue) {
+        operations.add(new JsonPatchOperation(JsonPatchOperationKind.ADD,
+            Objects.requireNonNull(path, "'path' cannot be null."), null,
+            Objects.requireNonNull(rawJsonValue, "'rawJsonValue' cannot be null.")));
+        return this;
+    }
+
+    /**
+     * Appends a "replace" operation to this JSON Patch document.
+     * <p>
+     * See <a href="https://tools.ietf.org/html/rfc6902#section-4.3">JSON Patch replace</a> for more information.
+     *
+     * <p><strong>Code Samples</strong></p>
+     *
+     * {@codesnippet com.azure.core.experimental.jsonpatch.JsonPatchDocument.appendReplace#String-String}
+     *
+     * @param path The path to replace.
+     * @param rawJsonValue The raw JSON value to use as the replacement.
+     * @return The updated JsonPatchDocument object.
+     * @throws NullPointerException If {@code path} or {@code rawJsonValue} is null.
+     */
+    public JsonPatchDocument appendReplace(String path, String rawJsonValue) {
+        operations.add(new JsonPatchOperation(JsonPatchOperationKind.REPLACE,
+            Objects.requireNonNull(path, "'path' cannot be null."), null,
+            Objects.requireNonNull(rawJsonValue, "'rawJsonValue' cannot be null.")));
+        return this;
+    }
+
+    /**
+     * Appends a "copy" operation to this JSON Patch document.
+     * <p>
+     * See <a href="https://tools.ietf.org/html/rfc6902#section-4.5>JSON Patch copy</a> for more information.
```

Comment:
Missing a closing double-quote here.
```suggestion
     * See <a href="https://tools.ietf.org/html/rfc6902#section-4.5">JSON Patch copy</a> for more information.
```

Reply:
Fix in the last commit